linebot/api.py

```diff
@@ -124,6 +124,221 @@ def push_message(self, to, messages, timeout=None):
+
+        Uploads and attaches an image to a rich menu.
+
+        :param str rich_menu_id: IDs of the rechmenu
+        :param str path_to_image: path to the image to upload,
+                                  only allow jpeg or png due to content type limitation
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        """
+        # check content type by file header, and add it into header dict
+        with open(path_to_image, 'rb') as image:
+            image_content = image.read()
+
+            if image_content[0:3] == b'\xff\xd8\xff':
+                self.headers['Content-Type'] = 'image/jpeg'
+            elif image_content[0:8] == b'\x89PNG\r\n\x1a\n':
+                self.headers['Content-Type'] = 'image/png'
+
+        # send request
+        self._post(
+            '/v2/bot/richmenu/{rich_menu_id}/content'.format(rich_menu_id=rich_menu_id),
+            data=open(path_to_image, 'rb').read(),
+            timeout=timeout
+        )
+
+        # remove added header to avoid side effect in other apis
+        del self.headers['Content-Type']
+
+    def get_rich_menu_id_of_user(self, user_id, timeout=None):
+        """Call get rich menu ID of user API.
+
+        https://developers.line.me/en/docs/messaging-api/reference/#get-rich-menu-id-of-user
+
+        Gets the ID of the rich menu linked to a user.
+
+        :param str user_id: IDs of the user
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        """
+        response = self._get(
+            '/v2/bot/user/{user_id}/richmenu'.format(user_id=user_id),
+            timeout=timeout
+        )
+
+        return RichMenu.new_from_json_dict(response.json)
+
+    def unlink_rich_menu_from_user(self, user_id, timeout=None):
+        """Call unlink rich menu from user API.
+
+        https://developers.line.me/en/docs/messaging-api/reference/#unlink-rich-menu-from-user
+
+        Unlinks a rich menu from a user.
+
+        :param str user_id: ID of the user
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        """
+        self._delete(
+            '/v2/bot/user/{user_id}/richmenu'.format(user_id=user_id),
+            timeout=timeout
+        )
+
+    def download_rich_menu_image(self, rich_menu_id, timeout=None):
```

Comment:
Could you change to `get_rich_menu_image ` instead of `download_rich_menu_image ` ?
We would like to unify the names with other SDKs.

e.g.
https://github.com/line/line-bot-sdk-nodejs/blob/master/lib/client.ts#L165
https://github.com/line/line-bot-sdk-java/blob/master/line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingClient.java#L187